stable/anchore-engine/README.md

```diff
@@ -0,0 +1,653 @@
+  - name: PATH
+     value: /opt/rh/rh-postgresql96/root/usr/bin:/opt/app-root/src/bin:/opt/app-root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+  postgresPassword: <PASSWORD>
+  persistence:
+    size: 50Gi
+
+anchoreGlobal:
+  defaultAdminPassword: <PASSWORD>
+  defaultAdminEmail: <EMAIL>
+  openShiftDeployment: True
+```
+
+To perform an Enterprise deployment on OpenShift use the following anchore_values.yaml configuration
+
+*Note: Installs with chart managed PostgreSQL database. This is not a guaranteed production ready config.*
+```
+## anchore_values.yaml
+
+postgresql:
+  image: registry.access.redhat.com/rhscl/postgresql-96-rhel7
+  imageTag: latest
+  extraEnv:
+  - name: POSTGRESQL_USER
+    value: anchoreengine
+  - name: POSTGRESQL_PASSWORD
+    value: anchore-postgres,123
+  - name: POSTGRESQL_DATABASE
+    value: anchore
+  - name: PGUSER
+    value: postgres
+  - name: LD_LIBRARY_PATH
+    value: /opt/rh/rh-postgresql96/root/usr/lib64
+  - name: PATH
+     value: /opt/rh/rh-postgresql96/root/usr/bin:/opt/app-root/src/bin:/opt/app-root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+    postgresPassword: <PASSWORD>
+    persistence:
+      size: 20Gi
+
+anchoreGlobal:
+  defaultAdminPassword: <PASSWORD>
+  defaultAdminEmail: <EMAIL>
+  enableMetrics: True
+  openShiftDeployment: True
+
+anchoreEnterpriseGlobal:
+  enabled: True
+
+anchore-feeds-db:
+  image: registry.access.redhat.com/rhscl/postgresql-96-rhel7
+  imageTag: latest
+  extraEnv:
+  - name: POSTGRESQL_USER
+    value: anchoreengine
+  - name: POSTGRESQL_PASSWORD
+    value: anchore-postgres,123
+  - name: POSTGRESQL_DATABASE
+    value: anchore
+  - name: PGUSER
+    value: postgres
+  - name: LD_LIBRARY_PATH
+    value: /opt/rh/rh-postgresql96/root/usr/lib64
+  - name: PATH
+     value: /opt/rh/rh-postgresql96/root/usr/bin:/opt/app-root/src/bin:/opt/app-root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+    postgresPassword: <PASSWORD>
+    persistence:
+      size: 50Gi
+
+anchore-ui-redis:
+  password: <PASSWORD>
+```
+# Chart Updates
+See the anchore-engine [CHANGELOG](https://github.com/anchore/anchore-engine/blob/master/CHANGELOG.md) for updates to anchore engine.
+
+## Upgrading from previous chart versions
+A Helm post-upgrade hook job has been added starting with Chart version 1.6.0 - this job will shut down all previously running Anchore services and perform the Anchore DB upgrade process using a kubernetes job. The upgrade will only be considered successful when this job completes successfully. Performing an update after v1.6.0 will cause the Helm client to block until the upgrade job completes and the new Anchore service pods are started. To view progress of the upgrade process, tail the logs of the upgrade jobs `anchore-engine-upgrade` and `anchore-enterprise-upgrade`. These job resources will be removed upon a successful helm upgrade.
+
+## Chart version 1.6.0
```

Comment:
Let's truncate this to only include the upgrade from the stable repo to this one.

Reply:
So you want to remove all of the previous version upgrade sections? I was thinking of doing that as well since none of it is relevant when installing from this repo.